.github/workflows/release.yml

```diff
@@ -266,10 +282,30 @@ jobs:
 
           gh release create "${{ needs.plan.outputs.tag }}" --target "$RELEASE_COMMIT" $PRERELEASE_FLAG --title "$ANNOUNCEMENT_TITLE" --notes-file "$RUNNER_TEMP/notes.txt" artifacts/*
 
+  # Publish oxbuild to crates.io
+  publish-crates:
```

Comment:
we should pull this out into a separate workflow file and register it as a "publish job" in config